src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
+        ColorSupplier color = new ColorSupplier();
+        Ball ball = new Ball();
+        ball.setNumber(value);
+        ball.setColor(color.getRandomColor().toString());
+        return ball;

```

Comment:
1) Again, pay attention to https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level
2) One more point about using magic numbers in code https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants
3) When you create a constructor for a Ball class, don't forget about this point https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method